src/uct/ib/rdmacm/rdmacm_ep.h

```diff
@@ -9,20 +9,28 @@
 #include "rdmacm_iface.h"
 
 struct uct_rdmacm_ep {
-    uct_base_ep_t                      super;
-    void                               *priv_data;
-    ucs_list_link_t                    list_elem;       /* for the pending_eps_list*/
-    struct sockaddr_storage            remote_addr;
-    int                                is_on_pending;
-    uct_worker_cb_id_t                 slow_prog_id;
-    uct_rdmacm_ctx_t                   *cm_id_ctx;
+    uct_base_ep_t                          super;
+    uct_sockaddr_fill_priv_data_callback_t fill_priv_data_cb;
+    void                                   *fill_priv_data_arg;
+    uint32_t                               cb_flags;
+    ucs_list_link_t                        list_elem;   /* for the pending_eps_list */
+    struct sockaddr_storage                remote_addr;
+    int                                    is_on_pending;
```

Comment:
maybe put is_on_pending near cb_flags to reduce space

Reply:
why will it reduce space?

Reply:
because of padding